src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -190,7 +190,7 @@ private static async Task<ImmutableArray<ISymbol>> GetMemberSymbolsAsync(Documen
             {
                 case TNamespaceDeclarationSyntax namespaceNode:
                     var containerSymbol = (INamespaceSymbol)semanticModel.GetDeclaredSymbol(container, cancellationToken);
-                    return containerSymbol.GetMembers().SelectAsArray(m => (ISymbol)m);
+                    return containerSymbol.GetMembers().Where(el => el.IsFromSource()).SelectAsArray(m => (ISymbol)m);
```

Comment:
I think this extension should be available to use:

https://github.com/dotnet/roslyn/blob/09408ab8a29e03caddfb11f29328c05169ac7cde/src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs#L199-L225

```suggestion
                    return containerSymbol.GetMembers().SelectAsArray(m => m.IsFromSource(), m => (ISymbol)m);
```

Reply:
This extension is for `ImmutableArray`, while `GetMembers()` returns `IEnumerable`. I should have checked it before accepting

Reply:
Interesting. It seems that `INamespaceSymbol` shadows `INamespaceOrTypeSymbol.GetMembers` (which returns ImmutableArray).